core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/site/edit/SiteEditViewModel.kt

```diff
@@ -158,6 +175,16 @@ class SiteEditViewModel(
             }
         }
     }
+    fun onTogglePermission(flag: Long) {
```

Comment:
should be onToggleNavigationItem or something like this - these are not permission flags even though they follow the same pattern